mm2src/mm2_io/src/fs.rs

```diff
@@ -279,3 +279,18 @@ pub fn json_dir_entries(path: &dyn AsRef<Path>) -> Result<Vec<DirEntry>, String>
         })
         .collect())
 }
+
+/// Helper function to copy directories recursively
```

Comment:
We are calling `create_dir_all(dst)?;` at the beginning, let's make it more clear: 

```suggestion
/// Helper function to copy directories recursively. It assumes that the target directory does not exist; otherwise, it will fail.
```

Reply:
my bad, I got confused with file copying